src/language/operators.md

```diff
@@ -328,10 +335,15 @@ assert((value & bitmask) == 0x02); // AND
 assert((value & ~bitmask) == 0x20); // AND NOT
 assert((value | bitmask) == 0x2f); // OR
 assert((value ^ bitmask) == 0x2d); // XOR
+
 assert((value << 4) == 0x220); // Shift left
 assert((value >> 4) == 0x02); // Shift right
+
+// Shift right example that results in different behavior on web
+// due to negative operand:
```

Comment:
I guess this text is updated automatically because it's extracted from examples/misc/test/language_tour/operators_test.dart, or vice versa?

Reply:
Yep, we use an excerpt tool which pulls snippets from the `.dart` files where they are tested :)